documentation/memory/step3/exercice/server.js

```diff
@@ -0,0 +1,52 @@
+// node.js dependencies
```

Comment:
I'd change this exercise to a script instead of an HTTP Server. It will be more reasonable for newcomers.